### PR TITLE
[12.0] siret: FIX readonly status on company fields

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

File: l10n_fr_siret/models/company.py [hunk 1]
```diff
@@ -12,11 +12,11 @@ class ResCompany(models.Model):
 
     # siret field is defined in l10n_fr module on res.partner
     # with an unstored related field on res.company
-    siret = fields.Char(store=True)
+    siret = fields.Char(store=True, readonly=True)
     siren = fields.Char(
-        string='SIREN', related='partner_id.siren', store=True)
+        string='SIREN', related='partner_id.siren', store=True, readonly=False)
```
Comment (bealdav) [inline l10n_fr_siret/models/company.py:17]: For my information, I thought than write on related fields should be avoided causing perf issues.
Maybe it's a misunderstood on my side ?
Here if you click on partner m2o you can edit these fields with no real inconvenience.

Comment (alexis-via) [reply l10n_fr_siret/models/company.py:17]: I'm not aware of such write perf issues, but writing on res.company is not something that is done very often, so it's not really a problem.
Without this fix, the SIRET field on res.company was writable, but it wouldn't update the siren/nic field, and at the end the siret field of res.partner was empty. So the aim is to fix this bug, nothing else.

Comment (bealdav) [reply l10n_fr_siret/models/company.py:17]: ok

**Review by kryskool — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@legalsylvain the bot didn't merge it. Do you launch it again?

**Comment by legalsylvain** [comment]
Thanks pedro.

**Pull request merged**